docker-compose.benefit.yml

```diff
@@ -82,6 +82,14 @@ services:
       - 127.0.0.1:3000:3000
       - 127.0.0.1:3100:3100
 
+  mailhog:
+    image: 'mailhog/mailhog:latest'
```

Comment:
Let's try not use `:latest` as this might cause confusion in the future.

Replace with:

[mailhog/mailhog@sha256:8d76a3d4ffa32a3661311944007a415332c4bb855657f4f6c57996405c009bea](https://hub.docker.com/layers/mailhog/mailhog/latest/images/sha256-8d76a3d4ffa32a3661311944007a415332c4bb855657f4f6c57996405c009bea?context=explore)

Reply:
Also add:

`container_name: benefit-mailhog`

![image](https://user-images.githubusercontent.com/5328394/218671975-0487ce36-e0f0-43e4-ba1e-df0902e53699.png)